meson.build

```diff
@@ -0,0 +1,208 @@
+project(
+        'htop',
+	'c',
```

Comment:
It looks like it is mixing tabs and spaces here?

Reply:
oops, lemme fix that.